spacemacs/packages.el

```diff
@@ -2492,7 +2492,7 @@ displayed in the mode-line.")
 
 (defun spacemacs/init-volatile-highlights ()
   (use-package volatile-highlights
-    :init
+    :config
```

Comment:
Oh well... my suggestion above lacks the difference between `init` and `config` so all in all we have to explain the special case to the user...

But that's ok to me.